### DISCUSSION TITLE
Pricescale inside chart

### DISCUSSION THREAD

**Discussion by dollebrolle** [tradingview/lightweight-charts#713] [Q&A] [answered]
Hello, is it possible to have the price bound to the grid? 

Like in this image: 
![priceline](https://user-images.githubusercontent.com/22471710/111921128-232e6200-8a93-11eb-8ca0-78231ae4f3c6.PNG)

**Comment by timocov** [comment, marked as answer]
See https://github.com/tradingview/lightweight-charts/issues/208
